### ISSUE TITLE
[v1.1.0] Error: Cannot find module 'xcode'

### ISSUE THREAD

**Issue by iron9light** [kelvinhokk/cordova-plugin-localization-strings#11]
Hi,
I use v1.1.0 of your plugin and try to build only for android.
When I run `ionic cordova build android --prod --release`, I got `Error: Cannot find module 'xcode'`
My OS is windows 10 and I only installed Android SDK.
I only have one translate json: 'zh-CN.json'
```js
{
  "locale": {
    "android": ["zh-rCN"]
  },
  "config_android": {
    "app_name": "The App Name in Chinese"
  }
}
```

**Comment by iron9light** [comment]
I fixed this issue.
Here's my PR: #12 
Please check it.

**Comment by kelvinhokk** [comment]
have merged PR#12 into master.
